spec/orchestration_job_spec.rb

```diff
@@ -28,8 +32,8 @@ def perform
       args: ["some", "args"],
       job: job,
       workflow: [
-        {"klass" => "OrcTest::JobA", "args" => ["some"]},
-        {"klass" => "OrcTest::JobB", "args" => ["args"]}
+        {"klass" => "OrcTest::JobA", "args" => ["some"], "opts" => {}},
+        {"klass" => "OrcTest::JobB", "args" => ["args"], "opts" => {}}
```

Comment:
Should add specs showing how defining or passing in `job_options` shows up in the serialized workflow and if possible also how it's passed into calling `:set` per job class.